opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -841,6 +841,8 @@ namespace Opm
     computePerfRate(const Value& pressure_cell,
                     const Value& rs,
                     const Value& rv,
+                    const Value& rsw,
+                    const Value& rvw,
```

Comment:
These parameters appear to be unused.  Is that temporary, in which case they could be marked as `[[maybe_unused]]`, or permanent, in which case they could just be removed, or a programming error?

Reply:
To enable rvw for MSW had no priority so we did not go further into that and is not in our planning at the moment. The combination rvw and MSW will for now give a throw. The rvw is in particular relevant in combination with brine (to model salt precipitation) and brine is also not supported by MSW.

Reply:
Ok, I have removed the MSW part of the PR. Adding support for MSW + WATER + GAS needs to be done first and in a separate PR. Since MSW + CO2STORE is probably not the most used combination, this PR should be merged first IMO, and the MSW part added later.